.pre-commit-config.yaml

```diff
@@ -44,56 +44,3 @@ repos:
           # Use black profile for isort to avoid conflicts
           # see https://github.com/PyCQA/isort/issues/1518
           args: ["--profile", "black", --line-length=120]
-    - repo: local
```

Comment:
FYI - It's intended to remove pylint for now due to some issues. Diondra is looking into it.